crypto/err/evp.errordata

```diff
@@ -1,3 +1,4 @@
+EVP,138,BAD_DECRYPT
```

Comment:
Did you add this manually or did you run the `make_errors.go` utility? I'm surprised it doesn't sort them if you use the tool.

Reply:
This is the output of that utility. (This caused me a little trouble -- the CI is setup to fail unless this file *exactly* matches that output.)